src/Chunk.ts

```diff
@@ -67,6 +67,10 @@ export interface DynamicImportMechanism {
 	interopRight?: string;
 }
 
+const TIME_RENDER_MODULES = '- render modules';
+const TIME_RENDER_FORMAT = '- render format';
+const TIME_SOURCEMAP = '- sourcemap';
```

Comment:
Perhaps have the names without the `- ` and add this in the timers utility code rather?